src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -59,6 +70,26 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
         return list(getattr(client, func)(container_name))
     return completer
 
+def dir_path_completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-argument
+    client = _get_client(FileService, parsed_args)
+    share_name = parsed_args.share_name
+    directory_name = ''
+    if prefix:
+        directory_name = prefix
+        while directory_name:
+            if not client.exists(share_name, directory_name):
```

Comment:
Is there any way to avoid doing a service call in a loop? It'll likely be... slow...